docker/Dockerfile

```diff
@@ -16,9 +16,12 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 \
     -o flottbot ./cmd/flottbot
 
 FROM alpine:3.11
-RUN apk --no-cache add ca-certificates && mkdir config
+ENV USERNAME=flottbot
+ENV UID=900
```

Comment:
why 900? if i'm not mistaken, the first non-root UID in alpine is 1000?

Reply:
I chose that because I was using -S to create a "system" user, ie set not to be able to login and with shell set to /sbin/nologin. The range for system users is 100-1000, so I set a UID in that range. The alternative would be to set a UID > 1000 but that then risks clashing with ranges people user for actual users (as opposed to UIDs for system processes). UIDs below 1000 don't confer any special privileges as such.

I know some people prefer to set to an arbitrary very high UID (eg > 10000), so we could do that as an alternative? I think key things is to set a value (so that builds are reproducible) and document it. It's not going to work for everyone - whatever you choose, it'll clash with someone's UID allocation scheme :)

WRT group - I was just relying on the adduser behaviour of creating a group with matching name for the user, but I think being specific about it is probably a good plan - I'll re-do the pull.

Reply:
Ah, sorry - I was mistaken, adduser in alpine (which is busybox) doesn't create a group by default. So I'll definitely do that explicitly

Reply:
nice - thanks for the explanation and tweak! we'll check/test this out a bit to see what kind of impact it might have in some of our implementations, if any.

Reply:
Great! Let me know how you get on.